test.js

```diff
@@ -105,6 +105,18 @@ function setCouchDBEnv(container, next) {
     // container's port 80 is dynamically mapped to an external port
     var port = _.get(c,
       ['NetworkSettings', 'Ports', '5984/tcp', '0', 'HostPort']);
+
+    var skips = {
+      skip: [
+        'find all limt ten',
+        'find all skip ten limit ten',
+        'find all skip two hundred',
+        'isActual'
+      ],
+    };
```

Comment:
Should we not check for the user specified skipped tests? Like if I wanted to skip test `foo`, I can pass it as an env variable and then it should merge these two arrays into one before sending it out? i.e `process.env.LOOPBACK_MOCHA_SKIP = JSON.stringify(skips);`